.github/workflows/windows-msys2-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Windows/MSYS2 Build
+on: [workflow_dispatch]
+
+jobs:
+  msys2-mingw64:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup MSYS2
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install: |
+            mingw-w64-x86_64-gcc
+            mingw-w64-x86_64-autotools
+            mingw-w64-x86_64-gtk3
+            mingw-w64-x86_64-python-docutils
+            patch
+            rsync
+            unzip
+            dos2unix
+            mingw-w64-x86_64-nsis
+      - name: GTK-Bundle
+        run: |
+          mkdir -p geany_build/bundle/geany-gtk
+          cd geany_build/bundle/geany-gtk
+          bash $(cygpath -u "${GITHUB_WORKSPACE}")/scripts/gtk-bundle-from-msys2.sh -3
+      - name: CI-Build
+        run: |
+          DESTINATON=$(cygpath -u "${GITHUB_WORKSPACE}")/geany_build
+          VERSION=$(grep 'AC_INIT' configure.ac | sed -E 's/AC_INIT\(\[[^]]+\],\s*\[([^]]+)\].*/\1/')
```

Comment:
If you can afford it, a probably safer solution is to call autom4te:
`autom4te --no-cache --language=Autoconf-without-aclocal-m4 --trace AC_INIT:\$2 configure.ac`

Otherwise, let's make it a slightly more forgiving sed-only call:
`sed -rn 's/^AC_INIT[(][^,]*,\s*\[?([^],]*)\]?\s*,.*$/\1/p' configure.ac`

Reply:
applied suggested improvement, thanks